include/nuttx/can.h

```diff
@@ -44,7 +44,6 @@
 #define CAN_EFF_FLAG 0x80000000  /* EFF/SFF is set in the MSB */
 #define CAN_RTR_FLAG 0x40000000  /* Remote transmission request */
 #define CAN_ERR_FLAG 0x20000000  /* Error message frame */
-#define CAN_EVT_FLAG 0x10000000  /* Lower_half use this flags to report state switch event */
```

Comment:
Why to remove this flag?

Reply:
I see only these two instances in my tree:

```
./nuttx/include/nuttx/lin.h\068:#define LIN_EVT_FLAG              CAN_EVT_FLAG
./nuttx/include/nuttx/can.h\047:#define CAN_EVT_FLAG 0x10000000  /* Lower_half use this flags to report state switch event */
```

So, doesn't appear to be used, but if CAN_EVT_FLAG is removed then LIN_EVT_FLAG needs to be removed also.

I see only these two instances of LIN_EVT_FLAG in my tree:

```
./nuttx/include/nuttx/lin.h\054: * LIN_EVT_FLAG : Event message frame. Lower_half use this flag to report
./nuttx/include/nuttx/lin.h\068:#define LIN_EVT_FLAG              CAN_EVT_FLAG
```

Reply:
CAN_EVT_FLAG uses bit29.The CAN ID of the extended frame format (EFF) is 29 bits, ranging from 0 to 0x1FFFFFFF.CAN_EVT_FLAG is not good for the Extended frame format of can.
In CAN, CAN_EVT_FLAG is not currently used.Therefore, this identification bit was deleted in CAN.

LIN's ID only occupies 6 bits.Tlin has more usable bits, so consider a special FLAG that can not be shared with can.
LIN_EVT_FLAG of LIN is in use.Therefore, the following definition was added
'''
#define LIN_EVT_FLAG (1 << (LIN_ID_BITS + 3))